<?xml version="1.0"?>
<launch> 

<!--  <param name="/use_sim_time" value="true"/> -->
  <!-- Play the bagfile -->
 <node name="image_publisher" pkg="publish_images" type="pub_images" >  <!-- respawn="true" -->

    <remap from="/cam0/camera_info" to="/stereo/left/camera_info"/>
    <remap from="/cam0/image_raw" to="/stereo/left/image_raw"/>
    <remap from="/cam0/parameter_descriptions" to="/stereo/left/parameter_descriptions"/>
    <remap from="/cam0/parameter_updates" to="/stereo/left/parameter_updates"/>
    <remap from="/cam1/camera_info" to="/stereo/right/camera_info"/>
    <remap from="/cam1/image_raw" to="/stereo/right/image_raw"/>
    <remap from="/cam1/parameter_descriptions" to="/stereo/right/parameter_descriptions"/>
    <remap from="/cam1/parameter_updates" to="/stereo/right/parameter_updates"/>

</node>
  <node  ns="stereo" pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc"  output="screen"/>


  
</launch>
